### ISSUE TITLE
questions about "Cosine_Similarity"

### ISSUE THREAD

**Issue by sev777** [Huangxiusheng/DEM#1]
Hi, in Dynamics-aware Module, you choose use Cosine_Similarity between layers as the contribution score, but due to the Residual, the score should be larger than 0.9, could you please give some Cosine_Similarity results or other Analytical Experiments about Cosine_Similarity ?

**Comment by Huangxiusheng** [comment]
Thank you for your attention to our work. As mentioned in Figure 5 of the paper, we can observe the example results of cosine similarity, with a range of -0.45 to 0.25. If you have any other questions, please feel free to ask further.

**Comment by sev777** [comment]
> Thank you for your attention to our work. As mentioned in Figure 5 of the paper, we can observe the example results of cosine similarity, with a range of -0.45 to 0.25. If you have any other questions, please feel free to ask further.

Thanks, I misunderstanding about the input and output. The vector you used is the attention or mlp input and output, rahter than the layer.

**Comment by Huangxiusheng** [comment]
> Thanks, I misunderstanding about the input and output. The vector you used is the attention or mlp input and output, rahter than the layer.

Yes, if you have any other questions, please feel free to ask further
